### PR TITLE
ADO 2.0 Release 1

### PR DESCRIPTION
**Pull request by boyd-nguyen**
I meant to divide this into smaller pull requests, but because a lot was built on more recent changes, I thought it would save time to pool all changes into one PR.

The changes in this PR include:
1. **Resources** (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/milestone/3)
    - All metadata are now in a table with clearer heading (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/43).
    - There's a prominent section on how to use the resource + "See more" click-to-action (CTA) button (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/44).
    - Update QUT resources (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/45).
    - The category label for each resource is now colour coded.
    - Featured resource cards are now added to Resources landing page.
2. **Home page** (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/milestone/1)
    - Update and restyle Featured resource cards (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/38).
    - Add clearer signposts and CTAs on how to contact DO and how to explore ADO resources (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/39).
    - Move all logos to the bottom as that's not the most important content on this page (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/36).
3. **Header nav** (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/milestone/2)
    - Simplify the header nav to contain one-word topics only and move Resources tab closer to Home (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/40).
    - Remove tabs for Data governance Guide and ADO ecosystem. The Data governance guide can be accessed via Resources, and the ADO Ecosystem has been made a blog post (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/41).
4. **Blog**
    - Create a basic model for blog and converted the ADO ecosystem to a blog post (https://github.com/QUT-Digital-Observatory/australian_digital_observatory_site/issues/52).  
5. **Minor responsive design changes for mobile view**

### CODE REVIEW

File: Australian Digital Observatory/assets/static/custom.css
Comment (betsybookwyrm) [inline Australian Digital Observatory/assets/static/custom.css:1]: I haven't gone through this or the html files in depth but it looks great :) 

The bright green on the tags for the resources is slightly glary on my monitor, maybe tone it down just a tad (not too much, I do like a good green!), or maybe I need to calibrate my monitor.

I do really like that there's more colour, and I love the additional info table on the resources.

File: Australian Digital Observatory/templates/resources.html
Comment (betsybookwyrm) [inline Australian Digital Observatory/templates/resources.html:1]: good idea repeating this here as well as the front page.

**Review by betsybookwyrm — APPROVED** [review]
This is a great improvement, Boyd! A couple small requests in the review comments, but I'm happy with this as-is and you can address those requests (or not) separately.

Thanks for doing this :)

File: Australian Digital Observatory/content/blog/ado-ecosystem/contents.lr
[the remark this answers is not in the document]
Comment (boyd-nguyen) [reply Australian Digital Observatory/content/blog/ado-ecosystem/contents.lr:1]: That's a great point! I'll action that

File: Australian Digital Observatory/content/blog/contents.lr
[the remark this answers is not in the document]
Comment (boyd-nguyen) [reply Australian Digital Observatory/content/blog/contents.lr:1]: I'll have a look. There maybe a Lektor plugin somewhere

File: Australian Digital Observatory/content/resources/adored/contents.lr
[the remark this answers is not in the document]
Comment (boyd-nguyen) [reply Australian Digital Observatory/content/resources/adored/contents.lr:1]: I'm not sure either. I removed it under the impression that it was broken, so I might confirm with Marissa about this.

**Pull request merged**